src/adb/exec.js

```diff
@@ -0,0 +1,15 @@
+let path = require('path');
+let exec = require('mz/child_process').exec;
+let debug = console.log.bind(console, '[adb/exec]');
+let flag = (option, value) => value ? ` ${option} ${value}` : '';
+let adbPath = path.resolve(__dirname, '..', '..', 'bin', 'adb', process.platform);
```

Comment:
nit: Also I try to keep the calls to require in ascii-sorted order to make for easy lookups